packages/flutter_tools/test/general.shard/migrations/swift_package_manager_integration_migration_test.dart

```diff
@@ -2980,15 +3272,16 @@ String migratedNativeTargetSection(
   SupportedPlatform platform, {
   bool missingPackageProductDependencies = false,
   bool withOtherDependency = false,
+  String? otherNativeTarget,
```

Comment:
I updated this method to allow for inserting an additional PBXNativeTarget. To keep the readability for the Runner target, I went with a StringBuffer, to allow for splitting it up, while preserving the sections.